docs/usermanual.md

```diff
@@ -544,3 +545,94 @@ To ensure consistent Markdown formatting run the following:
 # Use git-bash on Windows
 npm run prettier:text
 ```
+
+### Swagger
+
+In order to run Swagger, you need to execute the IoT Agent
+(https://github.com/telefonicaid/iotagent-ul/blob/master/docs/installationguide.md#usage) and then you can access to:
+
+```
+<server_host>:7896/api-docs
+```
+
+If you want to test the HTTP Protocol, two importants points:
```

Comment:
From this point on, is this really needed to have the swagger web working at api-docs/ ? Or do you refer to steps needed to run IOTA-UL in general?

Reply:
steps needed to run IOTA-UL in general

Reply:
In that case, I'd suggest removing all this part (i.e. from L559). Note that you already include a link to a document that explains how to run IOTA in L553. Repeating a long explanation about it here is a bit out of scope from my point of view.

Reply:
fixed in 2808c0b11be011d26b2a3c9e0077e1c4a22c056c